benchmarks/src/main/java/org/apache/druid/benchmark/datagen/BenchmarkSchemas.java

```diff
@@ -225,4 +225,32 @@
     );
     SCHEMA_MAP.put("rollo", rolloSchema);
   }
+
+  static { // simple schema with null valued rows, no aggs on numeric columns
+    List<BenchmarkColumnSchema> simpleNullsSchemaColumns = ImmutableList.of(
+        // dims
+        BenchmarkColumnSchema.makeZipf("stringZipf", ValueType.STRING, false, 1, 0.5, 1, 101, 1.5),
+        BenchmarkColumnSchema.makeDiscreteUniform("stringUniform", ValueType.STRING, false, 1, 0.25, 1, 100000),
+        BenchmarkColumnSchema.makeSequential("stringSequentialHalfNull", ValueType.STRING, false, 1, 0.5, 0, 1000),
+
+        // metrics
```

Comment:
Looks like they are not metrics?

Reply:
woops, i copied another section :cry: :+1: